### ISSUE TITLE
WCAG - More status messages required

### ISSUE THREAD

**Issue by jolevesq** [fgpv-vpgf/fgpv-vpgf#3848]
More Status messages need to be addded using ARIA to identify all dynamic content changes made from the map on the page. In a single-page application people who cannot see and use screen readers do not know that content has changed on the same page. E.g., page is updated with search results.

**Comment by milespetrov** [comment]
Waiting for feedback on related issue https://github.com/fgpv-vpgf/fgpv-vpgf/issues/3847

**Comment by dan-bowerman** [comment]
As we are pretty certain the cause and solution to this is the same as #3847, I would like to close this one as duplicate, and we will keep track of further discussion and development in #3847.
